public/js/gogs.js

```diff
@@ -1621,11 +1621,13 @@ $(document).ready(function() {
       $($(this).data("target")).removeClass("disabled");
     } else {
       $($(this).data("target")).addClass("disabled");
+      $($(this).data("uncheck")).prop("checked", false);
     }
   });
   $(".enable-system-radio").change(function() {
     $($(this).data("enable")).removeClass("disabled");
     $($(this).data("disable")).addClass("disabled");
+    $($(this).data("uncheck")).prop("checked", false);
```

Comment:
You're so talented to update this mess and make it work 😄 I hate JS. LOL

Reply:
Not my cup of tea either :laughing: ... anyway, thx